pkg/console/dashboard_panels.go

```diff
@@ -249,6 +257,75 @@ func getVIP() string {
 	return outStr
 }
 
+func syncNodeInfo(ctx context.Context, g *gocui.Gui) {
+	// sync info at the beginning
+	doSyncNodeInfo(g)
+
+	syncDuration := 30 * time.Second
+	ticker := time.NewTicker(syncDuration)
+	go func() {
+		<-ctx.Done()
+		ticker.Stop()
+	}()
+	for range ticker.C {
+		doSyncNodeInfo(g)
+	}
+}
+
+func doSyncNodeInfo(g *gocui.Gui) {
+	nodeIP := getNodeInfo()
+	g.Update(func(g *gocui.Gui) error {
+		v, err := g.View("nodeInfo")
+		if err != nil {
+			return err
+		}
+		v.Clear()
+		fmt.Fprintf(v, "Node Info: \n\n%s", nodeIP)
+		return nil
+	})
+}
+
+func getNodeInfo() string {
+	var (
+		cmd      string
+		address  string
+		hostname string
+		out      []byte
+		err      error
+		device   string
+	)
+
+	// find node hostname
+	cmd = `hostname | tr -d '\r\n'`
+	out, err = exec.Command("/bin/sh", "-c", cmd).Output()
+	hostname = string(out)
+	if err != nil || hostname == "" {
+		logrus.Warnf("node didn't have a hostname")
+		hostname = ""
```

Comment:
Is any chance the hostname empty and valid?
I am not sure how we could do that :p 
If the empty hostname is invalid, maybe we should do some error handling here?

Reply:
We should not change it.
Hostname should not be empty.
I think we could only show error msg in console or log.
hmm

Reply:
OK, looks like the only thing we could do is show the error msg.